tests/solids/bccMo_2x1x1_SOREP/CMakeLists.txt

```diff
@@ -80,14 +80,14 @@ if(QMC_COMPLEX)
 #Deterministic test
 
   if(NOT QMC_MIXED_PRECISION)
-    list(APPEND DET_BCC_MO_VMC_SCALARS "totenergy" "-136.88596684 0.000002")
+    list(APPEND DET_BCC_MO_VMC_SCALARS "totenergy" "-136.88580235 0.000002")
```

Comment:
I thought #4383 already applied a fix and updated reference values. I'm wondering why this file needs to be touched.

Reply:
The fix in #4383 did not touch SOECPotential.cpp, and that reference value didn't get updated.

Reply:
Clear. Thanks.